pkg/mcp/server/monitoring.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/stats/view"
+)
+
+const (
+	typeURL   = "typeURL"
+	errorCode = "code"
+	errorStr  = "error"
+	connectionID = "connectionID"
+)
+
+var (
+	TypeURLTag      tag.Key
+	ErrorCodeTag    tag.Key
+	ErrorTag        tag.Key
+	ConnectionIDTag tag.Key
+
+	// buckets are powers of 4
+	byteBuckets = []float64{1, 4, 16, 64, 256, 1024, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864, 268435456, 1073741824}
+
+	ClientCount = stats.Int64(
```

Comment:
I haven't checked, but I'm guessing your `lint` job failed because the exported vars here don't have comments. It will be redundant but I think it is required.